lib/coins.py

```diff
@@ -293,6 +295,15 @@ def electrum_header(cls, header, height):
     def deserializer(cls):
         return Deserializer
 
+    @classmethod
+    def daemon(cls):
+        return Daemon
+
+    @classmethod
+    def block_processor(cls):
+        return BlockProcessor
+
```

Comment:
Any reason to have this a function and not a class member variable?  It can't change as the function takes no args to deduce a change.